microsoft-edge/webview2/concepts/distribution.md

```diff
@@ -3,29 +3,82 @@ description: Distribution options when releasing an app using Microsoft Edge Web
+For more information about versioning, see [Versioning][ConceptsVersioning] and [Globals][ReferenceWin3209538WebviewIdl].  
+
+### Understand the WebView2 Runtime and installer (Preview)  
+
+
+Microsoft Edge Stable Channel may not be installed on all user machines where your application runs.  Instead of requiring that users install Microsoft Edge, your application may use the Evergreen WebView2 Runtime and Installer \(Preview\).  The WebView2 Runtime is a customized copy of the Microsoft Edge binaries that is used to run your WebView2 applications .  When the WebView2 Runtime is installed, users are not able to use it as a normal browser.  For example, there is no desktop shortcut, start menu entry, users are not able to open a browser window using the Runtime binaries, and so on.  All Evergreen WebView2 applications on the device may use a single Evergreen WebView2 Runtime installation.  
+
+Today during the preview, the Evergreen WebView2 Runtime and Microsoft Edge Dev Channel are updated at the same time and have the same build.  In the future during the Preview, the WebView2 Team plans to update the WebView2 Runtime and match the same build as the Microsoft Edge Beta Channel.  In the future when WebView2 reaches General Availability \(GA\), the WebView2 Team plans to update the WebView2 Runtime and match the same build as the Microsoft Edge Stable Channel.  After GA, applications should use the WebView2 Runtime in production.  
+
+> [!IMPORTANT]
+> Do not ship WebView2 applications in production during the preview.  
+
+The WebView team recommends the following workflow to ensure the Evergreen WebView2 Runtime is available.  
+
+1.  Download the latest [Evergreen WebView2 Runtime Installer][Webview2Installer].  
+1.  Include the installer in your application installer or updater.  
+1.  During your application installation or update, check if the Evergreen WebView2 Runtime is already installed on the user machine.  If not, the application invokes the installer to install the runtime.  
+
+Depending on your scenario, you may need to change the above workflow.  For example, your application installer may download the Evergreen WebView2 Runtime Installer instead of including it in your application package.  
+
+> [!NOTE]
+> Both the WebView2 Runtime and WebView2 Runtime installer are in preview.  The preview has a limited initial scope and is only available as a standalone, per-machine install on Windows 10 on x64.  In the future, support for Windows 7, x86, and ARM64 is planned.  
+
+### Best practices for using WebView2 Runtime and non-stable Microsoft Edge channels  
+
+Consider the following recommendations during the preview.  
+
+1.  Make sure to use the [Evergreen WebView2 Runtime and Installer ][Webview2Installer] to develop or test your packaging and distribution pipeline.  In the future, your production application must include the installer.  
```

Comment:
`Must` is a strong word and not exactly true. I'd say is advised or should.

Reply:
Updated.  
```md
1.  Make sure to use the [Evergreen WebView2 Runtime and Installer][Webview2Installer] to develop or test your packaging and distribution pipeline.  In the future, your production application should include the installer.  
```